024-DoYouEvenSynapse/README.md

```diff
@@ -0,0 +1,34 @@
+# What The Hack - Synapse What The Hack
```

Comment:
Please make the title reflect the actual title.

"What The Hack: Do You Even Synapse?"

Reply:
Amended